src/nvim/window.c

```diff
@@ -676,7 +676,7 @@ void win_set_minimal_style(win_T *wp)
   }
 
   // signcolumn: use 'auto'
-  if (wp->w_p_scl[0] != 'a') {
+  if (wp->w_p_scl[0] != 'a' || STRLEN(wp->w_p_scl) == 8) {
```

Comment:
lets make it >= to forecome eventual support of ten column signcolumn :)

Reply:
Consider it done 😄